src/nupic/frameworks/opf/two_gram_model.py

```diff
@@ -32,16 +32,17 @@
 
 
 class TwoGramModel(model.Model):
-  """Two-gram benchmark model."""
+  """
+  Two-gram benchmark model.
+
+  :param inferenceType: (:class:`nupic.frameworks.opf.opf_utils.InferenceType`)
+  :param encoders: a dict of dicts, eventually sent to
+         :meth:`~nupic.encoders.multi.MultiEncoder.addMultipleEncoders` (see
+         there for details).
```

Comment:
Do you mean, see  method docs? (`addMultipleEncoders `) Or is there a missing hyperlink?